mitxgraders/formulagrader.py

```diff
@@ -538,7 +540,7 @@ def check_response(self, answer, student_input, **kwargs):
             if result['ok'] is True or result['ok'] == 'partial':
                 self.post_eval_validation(student_input, used_funcs)
             return result
-        except (StudentFacingError, InvalidInput):
+        except (StudentFacingError):
```

Comment:
Thanks for the feedback, I'll try to address comments in a bit.

**Proposal:** I want to do delete this entire try/catch sequence (and the similar one in FormulaGrader), consolidating them in `AbstractGrader.__call__`. The rule would be:
- if AbstractGrader.__call__ sees an MITXGraderError, it is displayed
- if AbstractGrader.__call__ sees any other error, a generic message is displayed instead.
(Well, modifying the above to account for `debug`).

Pro: Simple, unified way to handle errors, reinforces the pattern we've established of StudentFacingError/ConfigError. Backward-compatible with builtin graders.

Cons: Requires slightly more familiarity with the library in order to understand the code. E.g., in FormulaGrader, an unfamiliar person might be wondering why there aren't try/catch around. Also, could potentially mess up error messages for existing 3rd-party plugins. (Do any exist? I guess that Harvard one.)

Reply:
I'm happy to approve this. It would definitely simplify and consolidate the error handling.